contracts/dao/Implementation.sol

```diff
@@ -42,9 +42,10 @@ contract Implementation is State, Bonding, Market, Regulator, Govern {
 
         // add SushiSwap pool
         _state.provider.oracle = new Oracle(address(dollar()), Constants.getPairAddress());
+        oracle().setup(); // setup oracle
```

Comment:
this creates a new pair. I think we don't want this.

Reply:
not any more